__tests__/html/transcript.activityGrouping.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * @jest-environment ./__tests__/html/__jest__/WebChatEnvironment.js
+ */
+
+describe('transcript', () => {
```

Comment:
We are not testing each combo, otherwise, we could easily have many thousands of combos to test. We only chose combos which are more representative. E.g. some of the layouts were introduced as a test because it was buggy during development.

These tests are not based on the combo number. It's based on the actual bit flipped. Also, some bits are not binary.

The combo number is for human only, it's an easy way for:

1. Successively test different layout with a single click (using the plus/minus button)
1. When hitting a bug, it's easy to remember what combo is causing the issue, instead of remembering bunches of bits